src/components/oppgaver/OppgaveView.tsx

```diff
@@ -80,34 +81,34 @@ const harFilerMedFeil = (oppgaveElementer: OppgaveElement[]) => {
 const feilmeldingComponentTittel = (feilId: string, filnavn: string, listeMedFil: any) => {
     if (listeMedFil.length > 1) {
         return (
-            <div className="oppgaver_vedlegg_feilmelding_overskrift">
+            <SkjemaelementFeilmelding className="oppgaver_vedlegg_feilmelding_overskrift">
```

Comment:
Stilig, denne har jeg ikke sett før. 
Jeg ser den er her: https://design.nav.no/components/skjemagruppe/technical
Men de viser ikke hvordan det ser ut. Hva gjør den? 😊 (Spør for å lære 🤓📘)

Reply:
Den legger  på en standard styling (rød, bold) og en aria-live attributt som gjør at den leses opp av skjermleser hvis teksten i feilmeldingen endrer seg. Pluss koder teksten med p-tag istedet for div som var før) :)

Reply:
Stilig! 🎉